### PR TITLE
Create ru_ru.lang

### PR DESCRIPTION
**Pull request by PixelmonPRO**
Added Russian localization

### CODE REVIEW

File: src/main/resources/assets/mapfrontiers/lang/ru_ru.lang [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+mapfrontiers.actions=Действия
+mapfrontiers.create_frontier=Сделать\nтерриторию
+mapfrontiers.delete_frontier=Удалить\nтерриторию
+mapfrontiers.update_frontier=Обновить\nтерриторию
+mapfrontiers.update_settings=Обновить\nнастройки
+mapfrontiers.new_group_name=Новая Группа
+mapfrontiers.new_user=Новый Игрок
+mapfrontiers.new_user_error_uuid_size=UUID быть ровно 32 символа и может иметь разделители
+mapfrontiers.new_user_error_uuid_format=UUID неправильно сформирован
+mapfrontiers.new_user_error_user_repeated=Игрок уже состоит в другой группе
```
Comment (alejandrocoria) [inline src/main/resources/assets/mapfrontiers/lang/ru_ru.lang:51]: According to Google Translate it says "The user is already in _another_ group" but it should say "The user is already in _this_ group".

Comment (PixelmonPRO) [reply src/main/resources/assets/mapfrontiers/lang/ru_ru.lang]: Corrected :)

**Comment by alejandrocoria** [comment]
Please add `#PARSE_ESCAPES` as first line.

**Comment by alejandrocoria** [comment]
Thank you very much for the translation!
It will be in the next version, along with the fix to #1.

**Pull request merged**